### DISCUSSION TITLE
Helpful Links

### DISCUSSION THREAD

**Discussion by timvahlbrock** [Oeffis/Oeffis#9] [Ideas]
A place to collect existing projects, articles and other links that might be helpful during the course of the project.

**Comment by timvahlbrock** [comment]
https://github.com/marudor/bahn.expert

A Web App that queries a variety of HAFAS APIs.

**Comment by timvahlbrock** [comment]
https://github.com/planarnetwork/raptor

TypeScript Implementation of a public transit routing algorithm developed by Microsoft, in case we decide that we want to calculate routes ourselves.

**Comment by Paulomart** [comment]
https://github.com/public-transport

Multiple Clients and Tools for interacting with the HAFAS and other apis.
